Snowflake.Data.Tests/SFConnectionIT.cs

```diff
@@ -164,12 +167,21 @@ public void TestConnectWithoutHost()
         {
             using (IDbConnection conn = new SnowflakeDbConnection())
             {
-                String connStrFmt = "account={0};user={1};password={2}";
-                conn.ConnectionString = String.Format(connStrFmt, testConfig.account,
+                string connStrFmt = "account={0};user={1};password={2}";
+                conn.ConnectionString = string.Format(connStrFmt, testConfig.account,
                     testConfig.user, testConfig.password);
-                conn.Open();
-                Assert.AreEqual(conn.State, ConnectionState.Open);
-                conn.Close();
+                // Check that connection succeeds if host is not specified in test configs, i.e. default should work.
+                if (string.IsNullOrEmpty(testConfig.host))

```

Comment:
Connection attempt without specified host will not work if the AWS region is other than us-west-1. Added condition to the assert.

Reply:
I would like to make host a required test config.